packages/landing/src/components/debug.tsx

```diff
@@ -0,0 +1,79 @@
+import { Component, ComponentChild } from 'preact';
+import maplibre from 'maplibre-gl';
+import { Config } from '../config.js';
+
+export class Debug extends Component<{ map: maplibre.Map }> {
+  componentDidMount(): void {
+    setTimeout(() => this.props.map.resize(), 100);
+  }
+
+  render(): ComponentChild {
+    return (
+      <div className="debug">
+        <div className="debug__info">
+          <label className="debug__label">
```

Comment:
the label should be the title not the title and value

`<label>ImageId</label><div>{Config.map.layerId}</div>`

Reply:
Thanks fixed.